scipy/optimize/tests/test_linprog.py

```diff
@@ -1312,6 +1312,24 @@ def test_bug_10124(self):
                       method=self.method, options=self.options)
         _assert_success(res, desired_x=[10, -3], desired_fun=-22)
 
+    def test_bug_10349(self):
+        """
+        Test for tolerance issue on LPs with equality constraints
```

Comment:
This is a little more specific:
```suggestion
        Test for redundancy removal tolerance issue
```
The issue is really in the redundancy removal routine rather than the LP solvers. We don't need to mention that the problem has equality constraints because that's implied by "redundancy removal".